stripe-ui-core/src/main/java/com/stripe/android/uicore/elements/TextFieldDropdownUiItem.kt

```diff
@@ -0,0 +1,21 @@
+package com.stripe.android.uicore.elements
+
+import androidx.compose.runtime.Composable
+import com.stripe.android.uicore.strings.resolve
+
+internal data class TextFieldDropdownUiItem constructor(
```

Comment:
How does this relate to `CardBrandChoice : SingleChoiceDropdownItem`? That’s what `CardInputWidget` uses.